<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">KeePassX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="pageDatabase">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="DatabaseTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageSettings">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="SettingsWidget" name="settingsWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageWelcome">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QWidget" name="searchPanel" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSearchField" name="searchField" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="WelcomeWidget" name="welcomeWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Database</string>
    </property>
    <widget class="QMenu" name="menuRecentDatabases">
     <property name="title">
      <string>Recent databases</string>
     </property>
    </widget>
    <addaction name="actionDatabaseNew"/>
    <addaction name="actionDatabaseOpen"/>
    <addaction name="menuRecentDatabases"/>
    <addaction name="actionDatabaseSave"/>
    <addaction name="actionDatabaseSaveAs"/>
    <addaction name="actionDatabaseClose"/>
    <addaction name="separator"/>
    <addaction name="actionChangeMasterKey"/>
    <addaction name="actionChangeDatabaseSettings"/>
    <addaction name="separator"/>
    <addaction name="actionImportKeePass1"/>
    <addaction name="separator"/>
    <addaction name="actionLockDatabases"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEntries">
    <property name="title">
     <string>Entries</string>
    </property>
    <widget class="QMenu" name="menuEntryCopyAttribute">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Copy attribute to clipboard</string>
     </property>
    </widget>
    <addaction name="actionEntryNew"/>
    <addaction name="actionEntryClone"/>
    <addaction name="actionEntryEdit"/>
    <addaction name="actionEntryDelete"/>
    <addaction name="actionEntryCopyUsername"/>
    <addaction name="actionEntryCopyPassword"/>
    <addaction name="menuEntryCopyAttribute"/>
    <addaction name="actionEntryAutoType"/>
    <addaction name="actionEntryOpenUrl"/>
   </widget>
   <widget class="QMenu" name="menuGroups">
    <property name="title">
     <string>Groups</string>
    </property>
    <addaction name="actionGroupNew"/>
    <addaction name="actionGroupEdit"/>
    <addaction name="actionGroupDelete"/>
   </widget>
   <widget class="QMenu" name="menuExtras">
    <property name="title">
     <string>Extras</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEntries"/>
   <addaction name="menuGroups"/>
   <addaction name="menuView"/>
   <addaction name="menuExtras"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEntryNew"/>
   <addaction name="actionEntryEdit"/>
   <addaction name="actionEntryDelete"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDatabaseOpen">
   <property name="text">
    <string>Open database</string>
   </property>
  </action>
  <action name="actionDatabaseSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save database</string>
   </property>
  </action>
  <action name="actionDatabaseClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close database</string>
   </property>
  </action>
  <action name="actionDatabaseNew">
   <property name="text">
    <string>New database</string>
   </property>
  </action>
  <action name="actionEntryNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add new entry</string>
   </property>
  </action>
  <action name="actionEntryEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View/Edit entry</string>
   </property>
  </action>
  <action name="actionEntryDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete entry</string>
   </property>
  </action>
  <action name="actionGroupNew">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add new group</string>
   </property>
  </action>
  <action name="actionGroupEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit group</string>
   </property>
  </action>
  <action name="actionGroupDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete group</string>
   </property>
  </action>
  <action name="actionDatabaseSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save database as</string>
   </property>
  </action>
  <action name="actionChangeMasterKey">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Change master key</string>
   </property>
  </action>
  <action name="actionChangeDatabaseSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Database settings</string>
   </property>
   <property name="toolTip">
    <string>Database settings</string>
   </property>
  </action>
  <action name="actionImportKeePass1">
   <property name="text">
    <string>Import KeePass 1 database</string>
   </property>
  </action>
  <action name="actionEntryClone">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clone entry</string>
   </property>
  </action>
  <action name="actionEntryCopyUsername">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy username to clipboard</string>
   </property>
  </action>
  <action name="actionEntryCopyPassword">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Copy password to clipboard</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionEntryAutoType">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Perform Auto-Type</string>
   </property>
  </action>
  <action name="actionEntryOpenUrl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open URL</string>
   </property>
  </action>
  <action name="actionLockDatabases">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Lock databases</string>
   </property>
  </action>
  <action name="actionFindCaseSensitive">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Case Sensitive</string>
   </property>
  </action>
  <action name="actionFindCurrentGroup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Current Group</string>
   </property>
  </action>
  <action name="actionFindRootGroup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Root Group</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DatabaseTabWidget</class>
   <extends>QTabWidget</extends>
   <header>gui/DatabaseTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SettingsWidget</class>
   <extends>QWidget</extends>
   <header>gui/SettingsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WelcomeWidget</class>
   <extends>QWidget</extends>
   <header>gui/WelcomeWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSearchField</class>
   <extends>QWidget</extends>
   <header>gui/qocoa/qsearchfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
